lib/facade_converter.ts

```diff
@@ -176,6 +173,7 @@ export class FacadeConverter extends base.TranspilerBase {
     var fileName = decl.getSourceFile().fileName;
     fileName = this.getRelativeFileName(fileName);
     fileName = fileName.replace(/(\.d)?\.ts$/, '');
+    fileName = fileName.replace(FACADE_NODE_MODULES_PREFIX, '');
```

Comment:
Agreed. My last PR was a bit too targeted. I blame my insufficient knowledge of the tool; I learnt a great lot today ><